subsys/secure_storage/Kconfig.its_store

```diff
@@ -12,7 +12,7 @@ config SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_SETTINGS
 		&& $(dt_node_has_compat,$(dt_node_parent,$(DT_STORAGE_PARTITION)),fixed-partitions)
 	imply FLASH_MAP
 	imply NVS
-	select SETTINGS
+	depends on SETTINGS
```

Comment:
This will require changes to enable this (and `FLASH`) in the existing tests/samples under `tests/subsys/secure_storage/` and `samples/psa/`.

Reply:
I'd say rather add this as part of the already existing `depends on`, or split it into parts (e.g. putting `FLASH_HAS_DRIVER_ENABLED` separately), for consistency.

Reply:
The Kconfig options still need to be enabled in the samples, I think.